### PR TITLE
fix: add missing annotation and labels for collect-mrrc-params

### PR DESCRIPTION
**Pull request by ligangty**

### CODE REVIEW

File: tasks/collect-mrrc-params/collect-mrrc-params.yaml [hunk 1]
```diff
@@ -3,6 +3,11 @@ apiVersion: tekton.dev/v1
 kind: Task
 metadata:
   name: collect-mrrc-params
+  labels:
+    app.kubernetes.io/version: "3.1.0"
```
Comment (mmalina) [inline tasks/collect-mrrc-params/collect-mrrc-params.yaml]: We typically start with version 0.1.0, so I would suggest that. 1.0.0 would be acceptable as well.

Comment (ligangty) [reply tasks/collect-mrrc-params/collect-mrrc-params.yaml]: Fixed.

**Review by mmalina — APPROVED** [review]

**Comment by mmalina** [comment]
Just for reference, this came up when reviewing another PR: https://github.com/konflux-ci/release-service-catalog/pull/510#discussion_r1715313209

And it was just an omission when this task was originally added.

**Comment by mmalina** [comment]
Merging even without a successful e2e run, because this task is not even included in any pipeline yet.

**Comment by johnbieren** [comment]
Thanks for this @ligangty!

**Pull request merged**